crates/extractors/src/lib.rs

```diff
@@ -17,7 +17,7 @@ type Result<T> = std::result::Result<T, Error>;
 pub fn map_policy(policy: &flow::SerPolicy) -> doc::SerPolicy {
```

Comment:
please also add these fields to `flow::SerPolicy` so it's in sync with `doc::SerPolicy` and implement the policy mapping here.

Reply:
done